### ISSUE TITLE
playlist empty

### ISSUE THREAD

**Issue by cauerego** [MaxFour/Music-Player#88]
### Actual behaviour
the playlists are always empty, including favourites

### Expected behaviour
should be able to add songs to them!
 
### Steps to reproduce
1. create playlist
2. add song
3. it stays empty

### Environment data
Android version: 10 00WW_2_270_SP05

Device model: nokia 7.2 TA-1196

Stock or customized system: stock

Music-Player app version: 1.3.3 (and all versions i've used before)

oddly enough, got the same issue with vinyl https://github.com/AdrienPoupa/VinylMusicPlayer/issues/242

**Comment by LionsSharks15** [comment]
As far as I know, this is a problem with Android 10. I just got it on my Samsung Galaxy Tab A and now I can't add songs to playlists. 
Best workaround: **Download the Samsung Music app, if possible.** You can use it just for playlist management like I do.

**Comment by cauerego** [comment]
*i'll never ever use anything from samsung on purpose!*

funny enough android 11 is indeed not showing this issue on my pixel 3...

i wouldn't say it's "a problem with 10", nor would i wait android to fix it. but for sure it looks to be related!

**Comment by cauerego** [comment]
using [eon player](https://play.google.com/store/apps/details?id=qijaz221.github.io.musicplayer) there was an interesting development... it could create a favourites list of its own, but any other playlist, created by it or outside, would bring an error message and not work at all.

edit: and then, after a few minutes, that also stopped working with the same error message! 🤣

fuck android 10.

**Comment by Daksh777** [comment]
@cauerego It's because of introduction of scoped storage. The only solution is to use a custom database, check this issue which has been reported to Google issue tracker:
https://issuetracker.google.com/issues/147619577

**Comment by cauerego** [comment]
@Daksh777 makes sense... and it's also refreshing to finally find a player that do that! [retro music](https://play.google.com/store/apps/details?id=code.name.monkey.retromusic) (which you are part of, huh).

but android 11 also enforces saf and show no such issue, so that still doesn't feel like the whole story.
